Sources/StreamChatUI/Utils/UIImageView+Extensions.swift

```diff
@@ -1,12 +1,13 @@
 //
-// Copyright © 2020 Stream.io Inc. All rights reserved.
+// Copyright © 2021 Stream.io Inc. All rights reserved.
 //
 
 import Nuke
 import UIKit
 
 extension UIImageView {
-    func setImage(from url: URL, placeholder: UIImage? = nil) {
+    func loadImage(from url: URL?, placeholder: UIImage? = nil) {
+        guard let url = url else { image = nil; return }
         let options = ImageLoadingOptions(placeholder: placeholder)
         Nuke.loadImage(with: url, options: options, into: self)
```

Comment:
I remember we had big images in some attachments, and It looks that Nuke does not downsample the images by default. In order to do it, we should add: `processors: [ImageProcessors.Resize(size: imageView.bounds.size)]`. This is out of the scope of this PR, but it will probably improve the performance of the chat significantly 🤔

Reply:
Good idea @nuno-vieira , can you please put it to JIRA?